tools/llm_bench/task/text_generation.py

```diff
@@ -379,9 +377,9 @@ def run_text_generation_genai_with_stream(input_text, num, model, tokenizer, arg
     max_gen_tokens = DEFAULT_OUTPUT_TOKEN_SIZE if args['infer_count'] is None else args['infer_count']
     streamer.reset()
     gen_config = model.get_generation_config()
+    gen_config.rng_seed = args["seed"]
     gen_config.max_new_tokens = max_gen_tokens
     gen_config.num_beams = args["num_beams"]
-    gen_config.do_sample = False
```

Comment:
we need to revert it once GenerationConfig::validate is fixed, otherwise OV validation will validate multinomial instead of greedy after this change.

Reply:
But now, some of the models failed because their config have `top_p != 1.0`  but `llm_bench` forces `do_sample=False`

Reply:
PR https://github.com/openvinotoolkit/openvino.genai/pull/1473

Reply:
Returned back